test/model.test.js

```diff
@@ -4229,7 +4247,12 @@ describe('Model', function() {
       });
 
 
-      await Person.collection.drop();
+      await Person.collection.drop().catch(err => {
+        if (err.codeName === 'NamespaceNotFound') {
```

Comment:
Should mongoose, generally, ignore this error and return a fake result?
If I drop a collection and that connection does not exist, I wouldn't mind if I just receive a response saying that it's dropped.

What do you think? @vkarpov15

Reply:
@AbdelrahmanHafez we may consider that for a future release, but it's a pretty low priority.

Reply:
I agree it's a low priority.